tools/legal-review/engine/org.graalvm.python.python-language-23.1.0/copyright-ignore

```diff
@@ -0,0 +1,48 @@
+"Copyright (c) Oracle and/or its affiliates. Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.");
```

Comment:
shouldn't this line be included?

Reply:
Oracle uses roughly the following template for their copyright headers:
```
Copyright (c) <year_start>, <year_end>, Oracle and/or its affiliates.
```

Which means that at the end, there is a massive amount of the same copyright headers that differ just in the starting year or the ending year. I just tried to make sure that we have at least one of these copyrights in our notice and ignored the rest.